examples/ios/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+use bevy::prelude::{
+    shape, AddDefaultPlugins, App, Assets, Camera3dComponents, Color, Commands, FaceToward,
+    IntoQuerySystem, LightComponents, Mat4, Mesh, Msaa, PbrComponents, ResMut, StandardMaterial,
+    Transform, Vec3, WindowDescriptor,
+};
+use bevy_window::WindowMode;
+
+#[no_mangle]
+extern "C" fn main_rs() {
+    App::build()
+        .add_resource(WindowDescriptor {
+            vsync: true,
+            resizable: false,
+            mode: WindowMode::BorderlessFullscreen,
+            ..Default::default()
+        })
+        .add_resource(Msaa { samples: 4 })
+        .add_default_plugins()
+        .add_startup_system(setup.system())
+        .run();
+}
+/// set up a simple 3D scene
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // add entities to the world
+    commands
+        // plane
+        .spawn(PbrComponents {
+            mesh: meshes.add(Mesh::from(shape::Plane { size: 10.0 })),
+            material: materials.add(Color::rgb(0.1, 0.2, 0.1).into()),
+            ..Default::default()
+        })
+        // cube
+        .spawn(PbrComponents {
+            mesh: meshes.add(Mesh::from(shape::Cube { size: 1.0 })),
+            material: materials.add(Color::rgb(0.5, 0.4, 0.3).into()),
+            transform: Transform::from_translation(Vec3::new(0.0, 1.0, 0.0)),
+            ..Default::default()
+        })
+        // sphere
+        .spawn(PbrComponents {
+            mesh: meshes.add(Mesh::from(shape::Icosphere {
+                subdivisions: 4,
+                radius: 0.5,
+            })),
+            material: materials.add(Color::rgb(0.1, 0.4, 0.8).into()),
+            transform: Transform::from_translation(Vec3::new(1.5, 1.5, 1.5)),
+            ..Default::default()
+        })
+        // light
+        .spawn(LightComponents {
+            transform: Transform::from_translation(Vec3::new(4.0, 8.0, 4.0)),
+            ..Default::default()
+        })
+        // camera
+        .spawn(Camera3dComponents {
+            transform: Transform::new(Mat4::face_toward(
```

Comment:
We made breaking changes to this api since you made this pr :)

We can fix this compilation error by using this instead:

```rs
transform: Transform::from_translation(Vec3::new(-3.0, 5.0, 8.0))
    .looking_at(Vec3::default(), Vec3::unit_y()),
```

Reply:
Yup. I was noticing this.